tools/cuttlefish-host-image-installer/utils/download-ci-cf.sh

```diff
@@ -4,11 +4,11 @@
 
 set -o errexit
 
-URL=https://ci.android.com/builds/latest/branches/aosp-master/targets/aosp_cf_arm64_phone-userdebug/view/BUILD_INFO
+URL=https://ci.android.com/builds/latest/branches/aosp-master/targets/aosp_cf_arm64_only_phone-userdebug/view/BUILD_INFO
```

Comment:
Better to be as `aosp_cf_arm64_only_phone-trunk_staging-userdebug` in `aosp-main-throttled`. Since `aosp-master` is outdated about a year ago, we cannot download the latest one.

Reply:
Thanks for catching the bug. Yes, both the branch name and target name are wrong. Fixed.